playbooks/setup.yml

```diff
@@ -12,6 +12,7 @@
     - { role: nexcess.php, php_prefix: "php56" }
     - { role: nexcess.php, php_prefix: "php70" }
     - { role: nexcess.php, php_prefix: "php71" }
+    - { role: nexcess.php, php_prefix: "php72" }
```

Comment:
We 100% need this but installing 4 PHPs by default seems like much.  Each one adds 8 - 10 minutes of install time.  Can we get a consensus on what PHP version are actually needed by default for cloudhosts @rfxn ?

Reply:
I feel we have to have it there. Magento 2 has plans to support PHP 7.2 in an upcoming release and   wordpress supports it now. Not having 7.2 available will just cause tickets over the coming year like 7.1 does now on our el6 plans.